<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatTemplatesDlg</class>
 <widget class="QDialog" name="ChatTemplatesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit chat templates</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="templates_groupBox">
     <property name="title">
      <string>Chat Templates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="MyTableView" name="chatTemplatesTableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ctmsg_groupbox">
        <property name="title">
         <string>Chat Template</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="CTMsgLabel">
             <property name="text">
              <string>Template</string>
             </property>
             <property name="buddy">
              <cstring>CTMsgEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="CTMsgEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="CTVarButton">
           <property name="text">
            <string>&amp;Variables...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CTDefValButton">
           <property name="text">
            <string>Reset to Default Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CTDefAllValButton">
           <property name="text">
            <string>Reset All to Default Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>mytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChatTemplatesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChatTemplatesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
